mpt/impl/smtp/cassandra/src/test/java/org/apache/james/mpt/smtp/host/CassandraJamesSmtpHostSystem.java

```diff
@@ -109,7 +110,7 @@ protected GuiceJamesServer createJamesServer() throws Exception {
             .build();
 
         return new GuiceJamesServer(configuration)
-            .combineWith(CassandraJamesServerMain.CASSANDRA_SERVER_MODULE, CassandraJamesServerMain.PROTOCOLS, new ProtocolHandlerModule())
+            .combineWith(CASSANDRA_MODULE_AGGREGATE, new ProtocolHandlerModule())
```

Comment:
I think SMTP-only don't need quota plugins

Reply:
Shouldn't we play our integration test on a whole system?

Does it protect us from forgetting plugin registration in the future?

Reply:
In my understanding, this system is used only for smtp test without mailbox storage.
Quota is about storage so yes, it's an integration test, but not of the IMAP + SMTP combination AFAIK.